lib/css_parser/parser.rb

```diff
@@ -40,7 +39,8 @@ def initialize(options = {})
                   import: true,
                   io_exceptions: true,
                   rule_set_exceptions: true,
-                  capture_offsets: false}.merge(options)
+                  capture_offsets: false,
+                  user_agent: "Ruby CSS Parser/#{CssParser::VERSION} (https://github.com/premailer/css_parser)"}.merge(options)
```

Comment:
then use it here :D

Reply:
Funny that, now that you say it, seems pretty obvious haha